resources/tdm_gcc.rb

```diff
@@ -87,6 +87,19 @@
       overwrite true
     end
   end
+
+  # Patch time.h headers for compatibility with winpthreads.
+  # These patches were made for binutils 2.25.1 for 32-bit TDM GCC only.
```

Comment:
Are these patches something that should be fixed in the upstream product? At the very least it feels like a pointer to some more details on the issue this is fixing.